src/components/formGenerator/formInterface.ts

```diff
@@ -0,0 +1,13 @@
+export interface FormConfig {
```

Comment:
Nu puteai sa bagi interfetele astea direct in FormGenerator?

Reply:
Done. Mutat direct in FormGenerator cele 2 interfete.